### PR TITLE
Fix: Spinner is static on IE

### PR DESCRIPTION
**Pull request by kialysa**
bug: on IE, `svg animate` is not supported, and `Spinner` is static (does not move)

fix: determine browser with helper function, and switch to a div with css `keyframes`

others: determine color of spinner in order of `this.props.color` →`theme.primary` → `default color`

Checklist:
- [ ] Test added / Snapshots updated
- [ ] Documentation added / updated

JIRA: https://allthings.atlassian.net/browse/APP-2836

### CODE REVIEW

File: src/atoms/Spinner.jsx [hunk 1]
```diff
@@ -8,52 +39,61 @@ import PropTypes from 'prop-types'
...
-    <circle
-      cx="50"
-      cy="50"
-      r="40"
-      stroke="#369980"
-      fill="none"
-      strokeWidth="8"
-      strokeLinecap="round"
+
+const Spinner = ({ color = '#369980', size = 30 }) =>
```
Comment (janrembold) [inline src/atoms/Spinner.jsx]: please use ColorPalette from allthings/colors dependency

Comment (janrembold) [reply src/atoms/Spinner.jsx]: hopefully there is one that fits 🤔

Comment (kialysa) [reply src/atoms/Spinner.jsx]: good point! Asked michael, and he said ColorPalette.blue would be good 💁‍♀️
Will change.

Comment (kialysa) [reply src/atoms/Spinner.jsx]: changed, but the auto-generated Spinner.md now says `default: ColorPalette.blue`. Think this would look odd?

Comment (janrembold) [reply src/atoms/Spinner.jsx]: Hmmm, good question. Actually it is correct but it needs some "insides" for external developers that maybe don't know anything about ColorPalette. But I think having the right color code is worth this confusion

**Pull request merged**